ansible/roles/datadog/vars/main.yml

```diff
@@ -0,0 +1,37 @@
+dd_checks:
+  ruby:
+    init_config:
+    logs:
+      - type: file
+        path: "/home/deploy/apps/{{app_name}}/shared/log/production.log"
+        service: ruby
+        source: ruby
+        sourcecategory: sourcecode
+  process:
+    init_config:
+    instances:
+      - name: ssh
+        search_string: ["ssh", "sshd"]
+      - name: syslog
+        search_string: ["rsyslog"]
+        cpu_check_interval: 0.2
+        exact_match: false
+        ignore_denied_access: true
+
+datadog_enabled: "{{ dd_enabled | default(true) }}"
+datadog_config:
```

Comment:
You could add `default_country` to the `group_vars` files where `deploy_env` sits.